base/libgit2/merge.jl

```diff
@@ -80,7 +80,7 @@ function merge!(repo::GitRepo, anns::Vector{GitAnnotated},
     anns_size = Csize_t(length(anns))
     @check ccall((:git_merge, :libgit2), Cint,
                   (Ptr{Void}, Ptr{Ptr{Void}}, Csize_t,
-                   Ptr{MergeOptionsStruct}, Ptr{CheckoutOptions}),
+                   Ptr{MergeOptions}, Ptr{CheckoutOptions}),
```

Comment:
update should do fast forwards or pull with rebase, so should this function also be deleted and/or moved to a different repo?